luminoso_api/upload.py

```diff
@@ -104,6 +105,11 @@ def main():
         help="username (defaults to your username on your computer)")
     parser.add_argument('-p', '--password', default=None,
         help="password (you can leave this out and type it in later)")
+    parser.add_argument('-d', '--date-format', default='iso',
```

Comment:
Is `iso` the actual best default?  Is there a value that could preserve backwards compatibility?  (Not that I'm saying that automatically makes it better -- just that I'm unconvinced that ISO is actually the most likely format out in the wild.)